src/components/HearingList.js

```diff
@@ -9,6 +14,47 @@ import LabelList from './LabelList';
 import Label from 'react-bootstrap/lib/Label';
 import LoadSpinner from './LoadSpinner';
 import getAttr from '../utils/getAttr';
+import HearingsSearch from './HearingsSearch';
+import toLower from 'lodash/toLower';
+import upperFirst from 'lodash/upperFirst';
+
+
+const HearingListTabs = () =>
+  <Nav className="hearing-list__tabs" bsStyle="tabs" style={{width: '100%'}} activeKey="1" pullRight={1}>
+    <NavItem eventKey="3" disabled style={{width: '70%'}}/>
+    <NavItem eventKey="1" href="/home">Lista</NavItem>
+    <NavItem eventKey="2" title="Item">Kartta</NavItem>
+  </Nav>;
+
+const HearingListFilters = ({labels, handleChangeFilter, handleSort}) =>
+  <div className="hearing-list__filter-bar">
+    <FormGroup controlId="formControlsSelect" className="hearing-list__filter-bar-filter">
+      <ControlLabel><FormattedMessage id="sort"/></ControlLabel>
+      <FormControl componentClass="select" placeholder="select" onChange={(event) => handleSort(event.target.value)}>
+        <option value="-created_at"><FormattedMessage id="newestFirst"/></option>
+        <option value="created_at"><FormattedMessage id="oldestFirst"/></option>
+        <option value="-close_at"><FormattedMessage id="lastClosing"/></option>
+        <option value="close_at"><FormattedMessage id="firstClosing"/></option>
+        <option value="-open_at"><FormattedMessage id="lastOpen"/></option>
+        <option value="open_at"><FormattedMessage id="firstOpen"/></option>
+        <option value="-n_comments"><FormattedMessage id="mostCommented"/></option>
+        <option value="n_comments"><FormattedMessage id="leastCommented"/></option>
+      </FormControl>
+    </FormGroup>
+    {labels.length !== 0 && <FormGroup controlId="formControlsSelect" className="hearing-list__filter-bar-filter">
+      <ControlLabel><FormattedMessage id="chooseTopic"/></ControlLabel>
+      <FormControl componentClass="select" placeholder="select" onChange={(event) => handleChangeFilter(event.target.value)}>
```

Comment:
The whitespace is obvious one and easy to fix, but I have few questions about the "Move label search below topic search, so they will have common search box" from the picture:

It is not actually a search now, it's a filter. After thinking this through with Ville, it made more sense this way than the way it was in the original wireframes. The way it is implemented now is way way easier (read: less expensive) to implement and better value & less confusing for the user.

The problem with the original wireframe was, that the logic how it should was not clear at all. What if the user fills both search fields? Should it show hearings that match the Title AND Label or Title OR Label?

The other problem was, how the user knows which labels there actually are? Now this should be a less of a problem with the autocomplete endpoint (which did not exist at the time of first iteration), but it won't solve it completely. The user still has to make a guess instead of skimming the available options to find something interesting.

TL;DR:

What I suggest that we do, is keep the label thing logic as a filter like it is now. If the dropdown size is a problem, it can be changed in to a multiselect box as you mentioned earlier. And to prevent the user confusing it with the search keep it where it is.

If this is to be changed the way suggested in the picture I need the following to implement:

1. Specification for the logic the search should work:
    a. Should the hearings be shown with AND or OR logic if both fields are selected? 
    b. Should the filtering happen onChange or onSubmit (the former was very clunky when I tested with the Title search)?

2. Specification how the label search logic should work:
    a. When the customer writes to the search field, should API call be made onChange and the suggestions be shown for the customer?
    b. What happens when the customer selects a label, should the filtering happen then or onSubmit?
    c. Should the customer be able to select multiple labels? If yes, how and where should the selected labes be rendered?
    d. If multiple labels are selected, should they work with AND or OR logic?